docs/examples/swift/iOS-CocoaPods/ThemisSwift/ThemisSwift/Resources/Base.lproj/LaunchScreen.storyboard

```diff
@@ -22,12 +20,12 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Themis Example Swift" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="eDm-A7-tHW">
-                                <rect key="frame" x="44" y="97" width="287" height="21"/>
+                                <rect key="frame" x="44" y="77" width="287" height="21"/>
                                 <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                 <nil key="textColor"/>
                                 <nil key="highlightedColor"/>
                             </label>
-                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Copyright (c) 2015-2019 Cossack Labs. All rights reserved." textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="SDJ-TG-gQB">
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Copyright (c) 2015-2020 Cossack Labs. All rights reserved." textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="SDJ-TG-gQB">
```

Comment:
This isn't the only project with an outdated copyright line.

```
$ git grep -o 'Copyright (c) .* All rights reserved.'
docs/examples/Themis-server/Obj-C/WorkingWithThemisServer/WorkingWithThemisServer/Base.lproj/LaunchScreen.storyboard:Copyright (c) 2015-2019 Cossack Labs. All rights reserved.
docs/examples/Themis-server/swift/SwiftThemisServerExample/SwiftThemisServerExample/Base.lproj/LaunchScreen.storyboard:Copyright (c) 2015-2019 Cossack Labs. All rights reserved.
docs/examples/objc/iOS-CocoaPods/ThemisTest/ThemisTest/Resources/Base.lproj/LaunchScreen.xib:Copyright (c) 2015-2019 Cossack Labs. All rights reserved." textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" minimumFontSize="9" translatesAutoresizingMaskIntoConstraints="NO" id="8ie-xW-0ye" userLabel="Copyright (c) 2015-2017 Cossack Labs. All rights reserved.
docs/examples/objc/iOS-CocoaPods/ThemisTest/ThemisTestTests/ThemisTestTests.m:Copyright (c) 2015 Cossack Labs. All rights reserved.
docs/examples/swift/iOS-CocoaPods/ThemisSwift/ThemisSwift/Resources/Base.lproj/LaunchScreen.storyboard:Copyright (c) 2015-2019 Cossack Labs. All rights reserved.
```

Also, how about leaving only the date when the company assumes copyright, implying that it's still actual in the present? That would save us the trouble of doing or forgetting to do pointless updates every year.

Reply:
yeah, makes sense

Reply:
removed copyright label at all from all projects